packages/webapp/cypress/support/commands.ts

```diff
@@ -29,7 +29,9 @@
  */
 Cypress.Commands.add("login", (account) => {
     cy.get(`[data-testid="signin-nav-buttonsm"]`).first().click();
+    cy.wait(500);
     cy.get("#ual-box div").contains("Password").click();
+    cy.wait(500);
```

Comment:
For some reason, I've had quite a few test failures on the induction tests reporting that `#ual-box input[type="text"]` can't be found. These waits seem to have alleviated that, but I'm still not quite happy with this.

Reply:
yeah I had these waits in the past exactly because of that... I removed trusting the cy.get would wait and detect it as they should... but still... flakey...

Reply:
Looks like I need to keep these around